### PR TITLE
Modify instructions in Devices section(User Guide)

### PR DESCRIPTION
**Pull request by HalynaDumych**
Modify instructions in the **Devices** section of the User Guide and
the  **Devices** section of the navigation file.

Relates-to: OTA-4584

Signed-off-by: Halyna Dumych <ext-halyna.dumych@here.com>

### CODE REVIEW

File: ota-plus-web/docs/modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -2,11 +2,10 @@
 .Get started
 * xref:index.adoc[Introduction] 
 * xref:get-access.adoc[Get access]
-// TODO * xref:switch-orgs.adoc[Switching Organizations]
 * xref:ota-workflow-campaign-manager.adoc[OTA update workflow]
 
 .Devices
-* xref:manage-device-prov.adoc[Provisioning]
+* xref:manage-device-prov.adoc[Whta is device provisioning?]
```
Comment (ammaboateng) [inline ota-plus-web/docs/modules/ROOT/nav.adoc]: Typo here.  I think you meant `What` instead.

Comment (HalynaDumych) [reply ota-plus-web/docs/modules/ROOT/nav.adoc]: Yep, thank you!
Corrected

**Comment by HalynaDumych** [comment]
> This isn't critical but I think the `Primary Control Unit` should be capitalized. But if it isn't capitalized in other areas of the documentation then it might be ok.

@ammaboateng 
It's a good point, but the phrase is usually not capitalized both in the documentation and in the app.

**Comment by ammaboateng** [comment]
@HalynaDumych, in that case, it's alright.

**Comment by ammaboateng** [comment]
LGTM

File: ota-plus-web/docs/modules/ROOT/pages/create-provisioning-key.adoc [hunk 2]
```diff
@@ -2,16 +2,23 @@
...
+3. In the *Add new key* dialog box, specify the key description, select its period of validity, and then click **Add key**.
++
+[.lightbackground]
+image::img::add_new_key.png[image,400]
 
-image::img::screenshot_provisioning_key_2.png[]
+The .zip file is added to the list of credentials. The file contains the provisioning key and credentials that your build system uses to publish images. You do not need to unzip the file. Save it somewhere on your computer. You will need the file when you set up your Yocto build or configure the {product-name} client.
 
-It comes as a zip file containing a provisioning key and credentials for your build system to publish images. You don't need to unzip it; just save it somewhere. You'll need it when you set up your Yocto build or configure the {product-name} client.
+For more information on device provisioning, see the xref:ota-client::client-provisioning-methods.adoc[Device Provisioning Methods, window="_blank"] section of the OTA Connect Client guide.
```
Comment (tkfu) [inline ota-plus-web/docs/modules/ROOT/pages/create-provisioning-key.adoc]:
```suggestion
For more a more in-depth technical description of device provisioning, see the xref:ota-client::client-provisioning-methods.adoc[Device Provisioning Methods] section of the OTA Connect Client guide.
```

Comment (HalynaDumych) [reply ota-plus-web/docs/modules/ROOT/pages/create-provisioning-key.adoc]: Changed

**Pull request merged**